app/models/green_lanes/category_assessment_search.rb

```diff
@@ -6,5 +6,5 @@ class GreenLanes::CategoryAssessmentSearch
   attr_accessor   :commodity_code,
                   :country
 
-  validates :commodity_code, presence: true, length: { minimum: 6, maximum: 10 }
+  validates :commodity_code, presence: true, length: { minimum: 10, maximum: 10, message: 'Please search declarable commodity.' }
```

Comment:
That message is a bit confusing and GDS guidelines are to not say 'Please'  - how about 'Enter a declarable 10 digit commodity code' - then the user knows how many digits its expecting?

Reply:
Done